flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/metrics/JobVertexMetricsHandler.java

```diff
@@ -28,15 +35,21 @@
  * {@code {"available": [ { "name" : "X", "id" : "X" } ] } }
  *
  * <p>If the query parameters do contain a "get" parameter, a comma-separated list of metric names is expected as a value.
- * {@code /metrics?get=X,Y}
+ * {@code /metrics?get=X,Y OR /metrics?get=X,Y&&subtasks=0-4,7-10}
  * The handler will then return a list containing the values of the requested metrics.
  * {@code [ { "id" : "X", "value" : "S" }, { "id" : "Y", "value" : "T" } ] }
  *
  * @deprecated This class is subsumed by {@link SubtaskMetricsHandler} and is only kept for backwards-compatibility.
  */
 @Deprecated
 public class JobVertexMetricsHandler extends AbstractMetricsHandler {
+	private final Logger log = LoggerFactory.getLogger(getClass());
```

Comment:
either add this as a protected field to the `AbstractMetricsHandler`, or change it to being static.